product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.3-Replace-a-value-of-the-incoming-message-using-enrich-mediator/src/test/java/org/wso2/carbon/ei/scenario/test/ReplaceElementsTest.java

```diff
@@ -141,7 +141,28 @@ public void replaceTargetDefinedThroughXpathBySourceInlineContent() throws IOExc
                 "urn:mediate", "replaceTargetDefinedThroughXpathBySourceInlineContent");
     }
 
-    //This test is to verify if payload can be modified by replacing body of payload using source defined inline through configuration registry
+    /** This test is to verify if payload can be modified by replacing body of payload using
```

Comment:
the comment should be in the next line from start of the block comment ("/*")
Eg: 
/** 
*. This test is to verify if payload can be modified by replacing body of payload using

Reply:
This is corrected.